### ISSUE TITLE
Use ReactModal in the LockScreen

### ISSUE THREAD

**Issue by satazor** [ipfs-shipyard/pm-idm#101]
## Description

By using ReactModal in the LockScreen, we are making sure that users cannot tab or focus elements outside the screen. It will also improve the overall accessibility.

## Acceptance Criteria

- [x] Use [ReactModal](https://github.com/reactjs/react-modal) in the LockScreen

**Comment by satazor** [comment]
Closing as this was already taken care of.
